### PR TITLE
Moved Refresh Filters button

### LINKED ISSUE #663: Add number to the reset filters button
The number in brackets should be:
`x = number_of_needs_selected + (any_profile_filter_selected ? 1 : 0)`

screen shot 2018-07-23 at 4 45 04 pm

### LINKED ISSUE #664: Move reset filters button on desktop
desktop:
screen shot 2018-07-23 at 4 45 04 pm

### PR DESCRIPTION
**Pull request by apmcguigan**
closes #663 and closes #664

### CODE REVIEW

**Comment by maxneuvians** [comment]
Here is PR review app: https://vac-poc-staging-pr-714.herokuapp.com/

File: components/profile_needs_selector.js [hunk 1]
```diff
@@ -41,6 +43,21 @@ export class ProfileNeedsSelector extends Component {
     open: false
   };
 
+  countSelected = () => {
+    let selectedProfileFilters = 0;
+    if (
+      this.props.selectedPatronType !== "" ||
```
Comment (smcmurtry) [inline components/profile_needs_selector.js:49]: Only need the 1st line, but nothing wrong with adding the other checks

File: components/profile_needs_selector.js [hunk 2]
```diff
@@ -60,15 +77,36 @@ export class ProfileNeedsSelector extends Component {
...
-          <Typography variant="title">{t("filters")}</Typography>
+          <Typography
+            variant="title"
+            className={classnames(classes.filterTitle)}
+          >
+            {t("filters")}{" "}
+            {(JSON.stringify(this.props.selectedNeeds) !== "{}" ||
+              this.props.patronType !== "") &&
+            pageWidth > 600 ? (
+              <Button
```
Comment (smcmurtry) [inline components/profile_needs_selector.js:95]: I played around with making this an `<a>`, which makes it small enough to not overflow in french. Maybe we can try that in a later PR.

**Review by smcmurtry — APPROVED** [review]
Great work!

**Pull request merged**